open-metadata-implementation/access-services/data-platform/data-platform-server/src/main/java/org/odpi/openmetadata/accessservices/dataplatform/views/ViewHandler.java

```diff
@@ -186,8 +189,9 @@ private EntityDetail addDerivedColumn(EntityDetail tableTypeEntity, String quali
             addRelationship(tableTypeEntity.getGUID(), derivedColumnEntity.getGUID(), Constants.ATTRIBUTE_FOR_SCHEMA);
             addRelationship(derivedColumnEntity.getGUID(), derivedColumn.getSourceColumn().getGuid(), Constants.SCHEMA_QUERY_IMPLEMENTATION);
             addRelationship(derivedColumnEntity.getGUID(), columnTypeEntity.getGUID(), Constants.SCHEMA_ATTRIBUTE_TYPE);
-            addRelationship(derivedColumnEntity.getGUID(), derivedColumn.getSourceColumn().getBusinessTerm().getGuid(), Constants.SEMANTIC_ASSIGNMENT);
-
+            for(BusinessTerm businessTerm : derivedColumn.getSourceColumn().getBusinessTerms()){
```

Comment:
In a future PR you should consider that in case the column exists, the business terms previously assigned could be different than the assigned business terms from the event.

Reply:
Ok thanks! I will mark it as a TODO in the next PR.